src/app/_components/_sharedcomponents/Cards/GameCard/GameCard.tsx

```diff
@@ -143,6 +319,19 @@ const GameCard: React.FC<IGameCardProps> = ({
 				</CardContent>
 			)}
 		</MuiCard>
+		{otherUpgradeCards.map((subcard, index) => (
+			<Box
+				key={subcard.uuid}
+				sx={{...styles.upgradeIconLayer,
+					backgroundImage: `url(${(cardUpgradebackground(subcard))})`,
+					bottom: `${index * 7 + 2}px`,
+					border: `1px solid ${subcard.selectable ? 'limegreen' : 'transparent'}`}}
+				onClick={() => upgradeClickFunction(subcard)}
```

Comment:
Let's comment out this onClick for now. I'm not sure that we want the upgrades to register click events here as most clicks on a card should be going to the attached cards. If we need to display options for clicking on an upgrade for targeting purposes, it is more likely we do that in a popup rather than on the board.

Reply:
sure thing! I was hoping we'd use the popup for this :D